packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -3256,6 +3366,82 @@ curl -v -X POST \
 ##### Activate Call Factor Response Example
 
 
+```json
+{
+  "expiresAt": "2015-11-03T10:15:57.000Z",
+  "status": "SUCCESS",
+  "sessionToken": "00Fpzf4en68pCXTsMjcX8JPMctzN2Wiw4LDOBL_9pe",
+  "_embedded": {
+    "user": {
+      "id": "00ub0oNGTSWTBKOLGLNR",
+      "passwordChanged": "2015-09-08T20:14:45.000Z",
+      "profile": {
+        "login": "dade.murphy@example.com",
+        "firstName": "Dade",
+        "lastName": "Murphy",
+        "locale": "en_US",
+        "timeZone": "America/Los_Angeles"
+      }
+    }
+  }
+}
+
+```
+
+#### Activate Email Factor
+
+Activates a `email` factor by verifying the OTP.  The request and response is identical to [activating a TOTP factor](#activate-totp-factor)
```

Comment:
English nits:  "a" should be "an"?  And perhaps "request and response _are_ identical"?

Reply:
fixed